iset.mm

```diff
@@ -58537,34 +58552,34 @@ ordinals implies excluded middle ( ~ ordsucunielexmid ).  (Contributed
     NUKBRZUMULSUJUQUIBTUAUQUMULBAUBUCUDUNULSUKABUEUFUGUH $.
 
   ${
-    $d A x $.  $d B x $.
+    $d A x y z $.  $d B x y z $.
+    $( If we remove a single element from a set with decidable equality then
+       put it back in, we end up with the original natural number.  This
```

Comment:
...with the original set

Reply:
Ooh good catch (copy paste error), thanks.

I've pushed up this change.